_posts/2015-06-03-team-offsite.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Team ePages 6 offsite event"
```

Comment:
Concerning "ePages 6" in the headline and in the text: I think the readers won't understand this. I would just call it "Team Green" (I think thats the right color?)